google/cloud/bigquery_v2/services/__init__.py

```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
```

Comment:
This whole `services` directory (and sub-directories) won't actually function correctly, as there's not a public endpoint for the models service. Let's exclude these from this PR.

Reply:
By "these", do you mean the models or everything that was re-generated under `bigquery_v2` with the microgenerator?

Reply:
Just the `services` directory. We still need the `types`.